weblate/machinery/microsoft.py

```diff
@@ -112,14 +122,15 @@ def download_languages(self):
 
     def download_translations(self, source, language, text, unit, user):
         """Download list of possible translations from a service."""
+        payload = _js.dumps([{'Text': text}])
+
         args = {
-            'text': text[:5000],
```

Comment:
Is length limitation gone from the API? I see you now use full text.

Reply:
You are correct, the limit for the translation is still [5000 chars](https://docs.microsoft.com/en-us/azure/cognitive-services/translator/request-limits).

Fixed now.